backend/src/server/routes/v2/project-router.ts

```diff
@@ -458,4 +459,44 @@ export const registerProjectRouter = async (server: FastifyZodProvider) => {
       return { collections: pkiCollections };
     }
   });
+
+  server.route({
+    method: "GET",
+    url: "/:projectId/certificate-templates",
+    config: {
+      rateLimit: readLimit
+    },
+    schema: {
+      params: z.object({
+        projectId: z.string().trim()
+      }),
+      response: {
+        200: z.object({
+          certificateTemplates: z.array(
```

Comment:
Return full certificate template object (no need to pick out `id` and `name`) for consistency with the `GET` certificate template by ID endpoint.

The endpoints should be predictable and similar as much as possible; this one should just be an array of the other one ideally.